### ISSUE TITLE
Any chance to have the respective amount of action pips consumed when swapping weapons sets in combat?

### ISSUE THREAD

**Issue by Cole-Fredrick** [Saibot393/enhancedcombathud-pf2e#42]
Right now if you're in combat and you swap weapon sets it would be nice to see the action pips reflect the amount of actions consumed if you were to swap to a worn weapon or even the amount of actions it would take to swap to a set where one of the weapons are stowed.

I may be blind but I don't see any option for this currently, any plans for implementation?

**Comment by Saibot393** [comment]
Yes that is something i planned for some time, never got around to implementing it though, maybe in the next version
